### PR TITLE
[10.0] [IMP] purchase_request: Add button to list purchase request line in purchase request

### PR DESCRIPTION
**Pull request by HviorForgeFlow**

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
code review

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional and code review

File: purchase_request/models/purchase_request.py [hunk 1]
```diff
@@ -110,6 +110,29 @@ def _track_subtype(self, init_values):
...
+    @api.depends('line_ids')
+    def _compute_line_count(self):
+        self.line_count = len(self.mapped('line_ids'))
+
+    @api.multi
+    def action_view_purchase_request_line(self):
+        action = self.env.ref(
+            'purchase_request.purchase_request_line_form_action').read()[0]
+        lines = self.mapped('line_ids')
+        if len(lines) > 1:
```
Comment (rousseldenis) [inline purchase_request/models/purchase_request.py:128]: @hveficent Why not using line_count? But that's detail

**Review by rousseldenis — APPROVED** [review]

File: purchase_request/__manifest__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "name": "Purchase Request",
     "author": "Eficent Business and IT Consulting Services S.L., "
               "Odoo Community Association (OCA)",
-    "version": "10.0.1.0.2",
+    "version": "10.0.1.1.2",
```
Comment (MiquelRForgeFlow) [inline purchase_request/__manifest__.py]: 10.0.1.1.0 instead

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_request/models/purchase_request.py (hunk 1 above)
Comment (HviorForgeFlow) [reply purchase_request/models/purchase_request.py:128]: I prefer to keep it this way :grin:

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**